MozillaFoundation/donate-wagtail
# FundraiseUp form update MVP

## Description:

Related PRs/issues: https://github.com/MozillaFoundation/foundation.mozilla.org/issues/10261

This PR updates the donate stack pages to use the javascript rendered FRU donation form.

This was done by using template overrides, so if we ever need to go back to our HTML form we can do so easily! 

This PR also removes the `logo_showcase` block from all templates, per request of our stakeholders.


## Steps to test:

1. Check this branch out
2. Run `inv new-db`
3. Take a look at http://localhost:8000/, and confirm that the landing page looks as expected.
4. Take a look at http://localhost:8000/campaign, and confirm that the campaign page looks as expected.
5. If you would like to test donations on either of these pages, append `?fundraiseupLivemode=no` to the URL!
6. If everything is looking as expected, testing is complete. Thank you! 🙌 


## Landing Page Screenshots:

**Landing Page Desktop:**
![Screenshot 2023-03-17 at 15-07-51 Donate today Donate to Mozilla](https://user-images.githubusercontent.com/18314510/226061575-f6ae5f95-c919-4215-9071-42a09317b91d.png)

**Landing Page Tablet:**
<img src="https://user-images.githubusercontent.com/18314510/226061597-04e74686-b034-47e1-934a-d71faefe328c.png" width="400">

**Landing Page Mobile:**
<img src="https://user-images.githubusercontent.com/18314510/225782316-7ed1dc4a-3bfb-4c10-b3ff-6e8c1a981b62.png" width="200">

## Campaign Page Screenshots:

**Campaign Page Desktop:**
![Screenshot 2023-03-16 at 17-18-35 It's Pi Day! Donate to Mozilla](https://user-images.githubusercontent.com/18314510/225782275-40c3c320-1786-4a6e-9648-c9509e9c906f.png)

**Campaign Page Tablet:**
<img src="https://user-images.githubusercontent.com/18314510/225782262-4feec784-84af-41b5-9e19-46de38514796.png" width="400">

**Campaign Page Mobile:**
<img src="https://user-images.githubusercontent.com/18314510/225782259-267155e9-a6b7-4b31-816c-a817f1365892.png" width="200">
